chainservice/builder.go

```diff
@@ -116,9 +117,9 @@ func (builder *Builder) BuildForSubChain() (*ChainService, error) {
 }
 
 // Build builds a chainservice
-func (builder *Builder) Build() (*ChainService, error) {
+func (builder *Builder) Build(opts ...factory.StateDBOption) (*ChainService, error) {
```

Comment:
Use `BuilderOption` instead

Reply:
I use `SetStopHeight` to keep consistent with other function like `SetActionPool`